airbyte-integrations/connectors/source-pivotal-tracker/source_pivotal_tracker/manifest.yaml

```diff
@@ -0,0 +1,1609 @@
+    - projects
+streams:
+  - type: DeclarativeStream
+    name: projects
+    primary_key:
+      - id
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $schema: http://json-schema.org/schema#
+        properties:
+          account_id:
+            type: number
+          atom_enabled:
+            type: boolean
+          automatic_planning:
+            type: boolean
+          bugs_and_chores_are_estimatable:
+            type: boolean
+          created_at:
+            type: string
+          current_iteration_number:
+            type: number
+          enable_following:
+            type: boolean
+          enable_incoming_emails:
+            type: boolean
+          enable_tasks:
+            type: boolean
+          has_google_domain:
+            type: boolean
+          id:
+            type: number
+          initial_velocity:
+            type: number
+          iteration_length:
+            type: number
+          kind:
+            type: string
+          name:
+            type: string
+          number_of_done_iterations_to_show:
+            type: number
+          point_scale:
+            type: string
+          point_scale_is_custom:
+            type: boolean
+          profile_content:
+            type: string
+          project_type:
+            type: string
+          public:
+            type: boolean
+          show_priority_icon:
+            type: boolean
+          show_priority_icon_in_all_panels:
+            type: boolean
+          show_story_priority:
+            type: boolean
+          start_time:
+            type: string
+          time_zone:
+            properties:
+              kind:
+                type: string
+              offset:
+                type: string
+              olson_name:
+                type: string
+            type: object
+          updated_at:
+            type: string
+          velocity_averaged_over:
+            type: number
+          version:
+            type: number
+          week_start_day:
+            type: string
+        type: object
+    retriever:
+      type: SimpleRetriever
+      requester:
+        type: HttpRequester
+        url_base: https://www.pivotaltracker.com/services/v5/
+        path: projects
+        http_method: GET
+        request_parameters: {}
+        request_headers: {}
+        authenticator:
+          type: ApiKeyAuthenticator
+          api_token: "{{ config['api_token'] }}"
+          inject_into:
+            type: RequestOption
+            inject_into: header
+            field_name: X-TrackerToken
+        request_body_json: {}
+      record_selector:
+        type: RecordSelector
+        extractor:
+          type: DpathExtractor
+          field_path: []
+      paginator:
+        type: DefaultPaginator
+        page_token_option:
+          type: RequestOption
+          inject_into: request_parameter
+          field_name: offset
+        pagination_strategy:
+          type: CursorPagination
```

Comment:
Should the `OffsetIncrement` strategy be used here?

Reply:
It is Cursor Pagination as described in docs and previous python version, the response gives next link and it is injected to path